doc/install.rst

```diff
@@ -213,8 +224,32 @@ Canopy and Anaconda for all supported platforms
 
 `Canopy
 <http://www.enthought.com/products/canopy>`_ and `Anaconda
-<https://store.continuum.io/cshop/anaconda/>`_ ships a recent
-version, in addition to a large set of scientific python library.
+<https://store.continuum.io/cshop/anaconda/>`_ both ship a recent
+version of scikit-learn, in addition to a large set of scientific python
+library for Windows, Mac OSX and Linux.
+
+Anaconda offers scikit-learn as part of its free distribution.
```

Comment:
@ogrisel if the question is whether scikit learn is available for free in canopy, that's only true for academic users.